src/ekore/anomalous_dimensions/unpolarized/time_like/as1.py

```diff
@@ -0,0 +1,117 @@
+# -*- coding: utf-8 -*-
+"""The following are the unpolarized time-like leading order Altarelli-Parisi splitting kernels."""
+
+import numba as nb
+import numpy as np
+from eko import constants
+
+
+
+@nb.njit(cache=True)
+def gamma_qq(N, s1):
+    """
+    Computes the LO quark-quark anomalous dimension
+    Implements Eqn. (B.3) from hep-ph/0604160
+
+    Input parameters
+    ----------------
+    N : Mellin moment (type: complex)
+    s1 : harmonic sum $S_{1}$ (type: complex)
+
+    Returns
+    -------
+    gamma_qq : LO quark-quark anomalous dimension $\gamma_{qq}^{(0)}(N)$ (type: complex)
+    """
+    result = constants.CF * (-3.0 + (4.0 * s1) - 2.0 / (N * (N + 1.0)))
+    return result
```

Comment:
If you are just computing the result in one shot, and you do not need to manipulate, you can also skip the assignment
```suggestion
    return constants.CF * (-3.0 + (4.0 * s1) - 2.0 / (N * (N + 1.0)))
```